packages/dds-components/src/components/ToggleButton/ToggleButton.module.css

```
@@ -50,6 +49,16 @@
   }
 }
 
+.small {
+  gap: var(--dds-spacing-x0-25);
+  padding: var(--dds-spacing-x0-5) var(--dds-spacing-x1);
+}
+
+.xsmall {
+  gap: var(--dds-spacing-x0-125);
+  padding: var(--dds-spacing-x0-25) var(--dds-spacing-x1);
```

Feil spacing på sidene.
```suggestion
  padding: var(--dds-spacing-x0-25) var(--dds-spacing-x0-5);
```